Formula/python@3.7.rb

```diff
@@ -95,19 +95,21 @@ def install
 
     args << "--without-gcc" if ENV.compiler == :clang
 
+    cpath    = []
```

Comment:
I didn't bother with `LIBRARY_PATH` here because it would always be empty for Python 3.7